### ISSUE TITLE
1.11.2 0.0.13-21 Client crash when trying to chisel quark Basalt

### ISSUE THREAD

**Issue by leagris** [Chisel-Team/Chisel#488]
##### Versions:

- Chisel-MC1.11.2-0.0.13.21
- forge-1.11.2-13.20.1.2393-universal
- [Other mods versions](http://rawgit.com/Beyond-Reality/Beyond-Realty-Farscapes/master/modlist.html?p=Beyond-Reality:%20Farscapes&v=1.1.0-r3)

##### Reproduce:

- Attempt to chisel Quark Basalt

#### Crash

[Complete Crash Report](https://pastebin.com/yEBNq16z)

##### Stack trace:

```
    at team.chisel.common.inventory.SlotChiselSelection.func_190901_a(SlotChiselSelection.java:68)
    at net.minecraft.inventory.Container.func_184996_a(Container.java:303)
    at team.chisel.common.inventory.ContainerChisel.func_184996_a(ContainerChisel.java:93)
    at net.minecraft.client.multiplayer.PlayerControllerMP.func_187098_a(PlayerControllerMP.java:551)
    at net.minecraft.client.gui.inventory.GuiContainer.func_184098_a(GuiContainer.java:632)
    at net.minecraft.client.gui.inventory.GuiContainer.func_73864_a(GuiContainer.java:391)
    at net.minecraft.client.gui.GuiScreen.func_146274_d(GuiScreen.java:544)
    at net.minecraft.client.gui.GuiScreen.func_146269_k(GuiScreen.java:512)
```

##### References:

https://github.com/Beyond-Reality/Beyond-Realty-Farscapes/issues/39

**Comment by Crumbology** [comment]
It was using a regular diamond chisel in the gui, allowed me to put the block in the gui, then as soon as I clicked on raw basalt, ctd

**Comment by RedlorMC** [comment]
MC: 1.11.2
Forge: 13.20.0.2313
Chisel: 1.11.2-0.0.13.20

similar stack trace being encountered using iron chisel to change a stack of blocks in chisel inventory.

**Comment by tterrag1098** [comment]
This was fixed by 9d5c99beb84115cd0633b33f155619b2636ad817
